packages/sysutils/busybox/scripts/init

```diff
@@ -591,14 +654,14 @@
             echo "zero-sized .md5 file..."
             MD5_FAILED="1"
           else
-            sed  -i 's#target#/storage/.update#g' "$UPDATE_DIR/${UPDATE_KERNEL}.md5"
-            sed  -i 's#target#/storage/.update#g' "$UPDATE_DIR/${UPDATE_SYSTEM}.md5"
+            sed "s#target#$UPDATE_DIR#g" "$UPDATE_DIR/${UPDATE_KERNEL}.md5" >"$UPDATE_ROOT/${UPDATE_KERNEL}.check.md5"
```

Comment:
What if I try to downgrade using an img that doesn't include the .check.md5 files (eg. downgrade using 6.0.3.img)? Should we allow such a downgrade, but warn the user that no checks are possible? Or we should not allow the downgrade to start when the .check.md5 files are missing.